server/server.go

```diff
@@ -586,7 +586,19 @@ func (s *Server) configureAccounts() error {
 	// Check opts and walk through them. We need to copy them here
 	// so that we do not keep a real one sitting in the options.
 	for _, acc := range s.opts.Accounts {
-		a := acc.shallowCopy()
+		var a *Account
+		if acc.Name == globalAccountName {
+			a = s.gacc
+		} else {
+			a = acc.shallowCopy()
+		}
+		if acc.hasMappings() {
+			// For now just move and wipe from opts.Accounts version.
+			a.mappings = acc.mappings
+			acc.mappings = nil
```

Comment:
If there are no mappings at all this will not detect a change. However you are saying if they do exist but no changes where made a reload would force use to re-evaluate the account, correct?

Reply:
because the mappings in opts are set to nill the deepequal in reload.go is bound to detect a change.
```
func (s *Server) diffOptions(newOpts *Options) ([]option, error) {
	...
	for i := 0; i < oldConfig.NumField(); i++ {
	...
		if changed := !reflect.DeepEqual(oldValue, newValue); !changed {
			continue
		}
```

Reply:
But we nil out the sl and clients, and that was before this change. So that would mean we always no matter what reload accounts..